examples/chart/teleport-cluster/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: teleport-cluster
 apiVersion: v2
-version: 6.0.0
+version: "6"
```

Comment:
oh, we're using only the major version?
so any changes to the chart can only happen during major releases?

Reply:
Hmm, that's not the intention.

When `VERSION` is set to a proper version like `7.0.1`, the charts' `version` and `appVersion` will automatically be set to `"7.0.1"` as well - we'll bump the Helm chart versions along with every Teleport release we make, and we'll also bump the Teleport release number up when we make Helm chart changes.

The `"6"` here was intended as a special case for when `VERSION` has a `-dev` suffix (as it does on `master`). We should really be setting it to `7.0.0`, but we can't because the `quay.io/gravitational/teleport:7.0.0` Docker tag won't exist until we publish the release. As such, overriding the chart's version to `6` on `master` was a way to keep the chart working as intended for anyone who deploys it from that branch.

This is a bit confusing now I look back on it. I'm not sure how we can handle this better, though.  I could update the logic in `version.mk` to instead handle `-dev` suffixes on `VERSION` by just using the most recently released Teleport tag instead?

Reply:
Can we publish a `:7.0.0-dev` tag for docker images from `master`?
People using the chart from `master` would then get the true "bleeding edge" experience (both code and chart).

Alternatively, if we already conditioned users to use helm from master, maybe de-couple `version` from `appVersion` and only set the old image tag in `appVersion`?